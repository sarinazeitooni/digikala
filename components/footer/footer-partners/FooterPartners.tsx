import React from 'react';
import style from './style/footer-partners.module.scss';
import urls from "./urls/urls";
import { v4 as uuidv4 } from 'uuid';
const FooterPartners = ()=>{
    return(
        <div className={style['partners-container']}>
            {
                urls.map((item:string)=>{
                    return(
                        <div key={uuidv4()} className={style['partner']}>
                            <img src={item}/>
                        </div>
                    )
                })
            }
        </div>
    )
};
export default FooterPartners;