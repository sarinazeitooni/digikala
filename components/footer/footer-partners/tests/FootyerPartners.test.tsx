import {shallow} from "enzyme";
import FooterPartners from "../FooterPartners";
describe("FooterPartners component" , ()=>{
    it("should render correctly" , ()=>{
        const container = shallow(<FooterPartners/>);
        expect(container).toMatchSnapshot();
    })
});