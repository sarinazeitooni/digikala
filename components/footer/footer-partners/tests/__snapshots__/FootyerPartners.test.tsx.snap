// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterPartners component should render correctly 1`] = `ShallowWrapper {}`;
